### DISCUSSION TITLE
Fake ps3 (Pro Controller) drops connection after being connected for 5-10s

### DISCUSSION THREAD

**Discussion by TarDuck** [nefarius/DsHidMini#95] [Q&A]
I, unfortunately, caved in and bought a fake ps3 controller, after doing everything in troubleshooting section of https://vigem.org/projects/DsHidMini/How-to-Install/ my controller worked through bluetooth but only for 5-10s where it would just drop the connection. I can automatically connect by pressing the middle bluetooth button but connection only stays for 5-10s I even tried to use it on a game and it functioned. Any help would be appreciated. Here is the link for the etl file 
[BthPS3.zip](https://github.com/ViGEm/DsHidMini/files/6571459/BthPS3.zip)

**Comment by Wiraginanjar** [comment]
i have a same problem, but here is my bthps3
[BthPS3.zip](https://github.com/ViGEm/DsHidMini/files/6887589/BthPS3.zip)

**Reply by Kanuan** [reply]
Controller is a `Pro Controller`, but it actually connects as expected, works but then disconnects after some seconds.
